velox/common/memory/tests/MemoryManagerTest.cpp

```diff
@@ -247,10 +262,10 @@ TEST_F(MemoryManagerTest, defaultMemoryManager) {
   ASSERT_EQ(managerB.numPools(), 0);
   ASSERT_EQ(
       managerA.toString(),
-      "Memory Manager[capacity 8388608.00TB alignment 64B usedBytes 0B number of pools 0\nList of root pools:\n\t__default_root__\n]");
+      "Memory Manager[capacity 512.00MB alignment 64B usedBytes 0B number of pools 0\nList of root pools:\n\t__default_root__\n]");
   ASSERT_EQ(
       managerB.toString(),
-      "Memory Manager[capacity 8388608.00TB alignment 64B usedBytes 0B number of pools 0\nList of root pools:\n\t__default_root__\n]");
+      "Memory Manager[capacity 512.00MB alignment 64B usedBytes 0B number of pools 0\nList of root pools:\n\t__default_root__\n]");
 }
```

Comment:
Can you add a failure test case? Thanks!

Reply:
Oh yeah sure, missed that